transmogrifier/sources/datacite.py

```diff
@@ -331,3 +337,13 @@ def generate_related_item_identifier_url(cls, related_item_identifier: Tag) -> s
         else:
             base_url = ""
         return base_url + related_item_identifier.string
+
+    @classmethod
+    def valid_content_types(cls, content_type_list: List[str]) -> bool:
+        """
```

Comment:
I would update the docstring to clarify that this may be overridden by subclasses that require content type validation, since the method here doesn't actually do that.